app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
```

Comment:
You should access class attribute through class name not `self`.
For this case it is `Person...`

Reply:
So, do I understand correctly that in ```__init__``` we always use ClassName.attribute ? We do never use this kind of construction ```self.__class__.attribute``` ?